GNUmakefile

```diff
@@ -263,7 +275,7 @@ pool-mgmt-image: cstor-pool-mgmt
 	@echo "--> cstor-pool-mgmt image "
```

Comment:
should we use the newly created variables here in this echo as well?
similarly for other echo statements

Reply:
Done!